autosklearn/automl.py

```diff
@@ -247,6 +247,11 @@ def __init__(
                 memory_limit=memory_limit,
             )
 
+        # If we got something callable for `get_trials_callback`, wrap it so SMAC
+        # will accept it.
+        if get_trials_callback is not None and callable(get_trials_callback):
```

Comment:
Does the SMAC callback itself also trigger this? If yes, can we maybe directly pass it through instead?

Reply:
It does but it shouldn't make a difference as the passed object will still get called. I left it out for simplicity. I'll add it back in though